docs/appendices/release-notes/5.0.0.rst

```diff
@@ -0,0 +1,205 @@
+.. _version_5.0.0:
+
+=============
+Version 5.0.0
+=============
+
+Released on 2022-07-11.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 4.0.2 or higher
```

Comment:
FunctionInfo was replaced by Signature in 4.2.0, but streaming of Signatures takes care of nodes <4.2.0, https://github.com/crate/crate/pull/12701